.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: Actions 😎
+name: Build and Tests
```

Comment:
Could you leave this as `actions` just without the emoji? it'll look more readable in this view: 
![image](https://user-images.githubusercontent.com/20756439/85184913-0958b800-b292-11ea-91ea-184b3dfcff0f.png)

Note that this will also affect the badge in the readme.